tests/integration/cli/test_run.py

```diff
@@ -1,19 +1,24 @@
 BAD_COMMAND = "not-a-name"
 
 
+def test_run_no_scripts_dir(ape_cli, runner, project):
```

Comment:
One thing I don't like about this is that it is not obvious what projects are skipped per test when looking at the test alone...
I am pretty sure using a very similar approach, I can make it work from a custom decorator, something like:

```python
@skip_projects(["interface", "script"])
def test_run_no_scripts_dir(ape_cli, runner, project):
  ...
```

if that seems interesting

Reply:
I think this would definitely help. I also think this decorator should throw if it doesn't match a valid project name to prevent ensuing madness

Reply:
Check out the decs!

I made two:

`skip_projects()`
and 
`skip_projects_except()`.

Because sometimes you literally want to test a weird part of just one project and that test would make zero sense anywhere else.  Now, it is quite intuitive to do this! I like this decorator approach a lot.